models/public/swin-tiny-patch4-window7-224/README.md

```diff
@@ -0,0 +1,84 @@
+# swin-tiny-patch4-window7-224
+
+## Use Case and High-Level Description
+
+The `swin-tiny-patch4-window7-224` model is a `tiny` version of the Swin Transformer image classification models pre-trained on ImageNet dataset. Swin Transformer is Hierarchical Vision Transformer whose representation is computed with shifted windows. Each patch is treated as a token and its feature is set as a concatenation of the raw pixel RGB values. By default, patch size is 4. The window parameter is responsible for the number of patches in each window, default value is 7. Stages of tiny version of model have 2, 2, 6, 2 layers respectively. Number of channels of the hidden layers in the first stage for tiny variant is 96.
```

Comment:
I think it is better to avoid words default, it is description of specific model and you need provide stuff related to it, there is no othrer models to compare parameters

Reply:
Updated